workflows/images/cellranger/README.md

```diff
@@ -0,0 +1,36 @@
+This folder contains a Dockerfile for the cellranger analysis.
+
+The cellranger archive component must be downloaded separately to comply with licensing, and should be placed in this folder.
+The current version of this file is `cellranger-4.0.0.tar.gz` and can be downloaded from [10X Genomics Website](https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest) after agreeing to their license terms.
+
+
+```
+docker build . -t scpca-cellranger:4.0.0
```

Comment:
This command doesn't work in the top of the repository (and this branch) as I expected it to. I get this warning: `unable to prepare context: unable to evaluate symlinks in Dockerfile path:` are there symlinks you've set that you were supposed to have worked? Or was I not supposed to run this command at the top of the repository?

Reply:
This will not work unless you have downloaded cellranger, I think. There should not be any symlinks, but I wonder if that is the error you get if a file is missing.